modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/edit_application_left_column.jsp

```diff
@@ -35,7 +35,7 @@ OAuth2Application oAuth2Application = oAuth2AdminPortletDisplayContext.getOAuth2
 		<aui:option label="private-key-jwt" value="private_key_jwt" />
 	</aui:select>
 
-	<aui:input helpMessage="json-web-key-set-help" label="JSON Web Key Set" name="jwks" style="min-height: 100px;" type="textarea" />
+	<aui:input cssClass=".min-height-100" helpMessage="json-web-key-set-help" label="JSON Web Key Set" name="jwks" type="textarea" />
```

Comment:
Hey @jgarcialr, I think you forgot the `.` here.

Reply:
done and change the className to another more meaningful